pkg/controller/nodenetworkconfigurationpolicy/conditions/manager.go

```diff
@@ -90,3 +97,59 @@ func (m *Manager) updateEnactmentConditions(
 		return nil
 	})
 }
+
+func (m *Manager) refreshPolicyConditions() error {
+
+	// On conflict we need to re-retrieve enactments since the
+	// conflict can denote that the calculated policy conditions
+	// are now not accurate.
+	return retry.RetryOnConflict(retry.DefaultRetry, func() error {
+
+		enactments := nmstatev1alpha1.NodeNetworkConfigurationEnactmentList{}
+		policyLabelFilter := client.MatchingLabels{"policy": m.policy.Name}
```

Comment:
If we do use labels, please namespace them: nmstate.io/policy or nmstate.io/configurationPolicy (configurationEnactment and state)